docs/getting-started.md

```diff
@@ -0,0 +1,629 @@
+            Movie Finder
+          </h1>
+          <p class="lead text-muted">
+            Let's go to the movies
+          </p>
+          <p class="lead">
+          </p>
+          <label for="searchTitle">Title:
+            <input type="textfield" id="searchTitle" name="searchTitle" />
+          </label>
+          <input type="button" name="Search" value="Search" onclick="handleSearch()" />
+```
+
+That's straightforward. But it still doesn't do very much. So let's go implement handleSearch().
+
+Here's where things get interesting. We can now call the content source we built in Step 2 to retrieve data for our search results. We can use the Fusion HTTP endpoint for fetching data from a content source. Let's check the data first to make sure we know what the data from our content source looks like. Hit this URL in your browser:
+
+http://localhost/pf/api/v3/content/fetch/movie-find?query={%22imdbID%22:%22tt0107290%22}
+
+This gives us back data like:
+
+```json
+{"Title":"Jurassic Park","Year":"1993","Rated":"PG-13","Released":"11 Jun 1993","Runtime":"127 min","Genre":"Adventure, Sci-Fi, Thriller","Director":"Steven Spielberg","Writer":"Michael Crichton (novel), Michael Crichton (screenplay), David Koepp (screenplay)","Actors":"Sam Neill, Laura Dern, Jeff Goldblum, Richard Attenborough","Plot":"Huge advancements in scientific technology have enabled a mogul to create an island full of living dinosaurs. John Hammond has invited four individuals, along with his two grandchildren, to join him at Jurassic Park. But will everything go according to plan? A park employee attempts to steal dinosaur embryos, critical security systems are shut down and it now becomes a race for survival with dinosaurs roaming freely over the island.","Language":"English, Spanish","Country":"USA","Awards":"Won 3 Oscars. Another 32 wins & 25 nominations.","Poster":"https://m.media-amazon.com/images/M/MV5BMjM2MDgxMDg0Nl5BMl5BanBnXkFtZTgwNTM2OTM5NDE@._V1_SX300.jpg","Ratings":[{"Source":"Internet Movie Database","Value":"8.1/10"},{"Source":"Rotten Tomatoes","Value":"91%"},{"Source":"Metacritic","Value":"68/100"}],"Metascore":"68","imdbRating":"8.1","imdbVotes":"784,720","imdbID":"tt0107290","Type":"movie","DVD":"10 Oct 2000","BoxOffice":"$45,299,680","Production":"Universal City Studios","Website":"http://www.jurassicpark.com/maingate_flash.html","Response":"True"}
+```
+
+Alright, so we know what the data looks like and how to retrieve it. It's straightforward from there to write a client-side call to this endpoint.
+
+Edit the handleSearch function to be:
+
+```javascript
+
+     const handleSearch = () => ({
+       // 1. Make an Ajax call to content source
+       // 2. Set data based on response
+       // 3. Re-render search results
+       const searchTerm = document.getElementById('searchTitle').value;
+
+       superagent
+         .get('/pf/api/v3/content/fetch/movie-search')
+         .query({ query: JSON.stringify({"text":searchTerm})})
```

Comment:
Not sure if they key in the stringified object should be `text` or not, it depends on what you are looking for in your content source obviously.